roles/base/tasks/redhat_tasks.yml

```diff
@@ -3,6 +3,7 @@
     update_cache: true
     name: '*'
     state: latest
+  ignore_errors: yes
```

Comment:
I get following error on two out of eight machines. It fails consistently on same two machine. Manually running "yum upgrade" goes thru.. Scavenging internet, it looks like there are few ansible bugs in this area like: https://github.com/ansible/ansible/issues/12248
If one of you can help me debug the issue, we can remove this line..
failed: [node2] => {"changed": true, "failed": true, "rc": 1, "results": ["Loaded plugins: fastestmirror\nLoading mirror speeds from cached hostfile\n * base: centos.arvixe.com\n * epel: mirrors.mit.edu\n * extras: repo.dimenoc.com\n * updates: mirror.trouble-free.net\nResolving Dependencies\n--> Running transaction check\n---> Package docker-engine.x86_64 0:1.9.1-1.el7.centos will be updated\n---> Package docker-engine.x86_64 0:1.10.1-1.el7.centos will be an update\n---> Package docker-engine-selinux.noarch 0:1.10.0-1.el7.centos will be updated\n---> Package docker-engine-selinux.noarch 0:1.10.1-1.el7.centos will be an update\n---> Package python-requests.noarch 0:2.6.0-1.el7_1 will be updated\n---> Package python-requests.noarch 0:2.7.0-1.el7 will be an update\n--> Finished Dependency Resolution\n\nDependencies Resolved\n\n================================================================================\n Package                Arch    Version                 Repository         Size\n================================================================================\nUpdating:\n docker-engine          x86_64  1.10.1-1.el7.centos     docker-main-repo  9.6 M\n docker-engine-selinux  noarch  1.10.1-1.el7.centos     docker-main-repo   28 k\n python-requests        noarch  2.7.0-1.el7             openstack-kilo     95 k\n\nTransaction Summary\n================================================================================\nUpgrade  3 Packages\n\nTotal download size: 9.7 M\nDownloading packages:\nDelta RPMs disabled because /usr/bin/applydeltarpm not installed.\n--------------------------------------------------------------------------------\nTotal                                              831 kB/s | 9.7 MB  00:11     \nRunning transaction check\nRunning transaction test\nTransaction test succeeded\nRunning transaction\n  Updating   : docker-engine-selinux-1.10.1-1.el7.centos.noarch             1/6 \n  Updating   : docker-engine-1.10.1-1.el7.centos.x86_64                     2/6 \n  Updating   : python-requests-2.7.0-1.el7.noarch                           3/6 \nerror: unpacking of archive failed on file /usr/lib/python2.7/site-packages/requests/packages/chardet: cpio: rename\nerror: python-requests-2.7.0-1.el7.noarch: install failed\nerror: python-requests-2.6.0-1.el7_1.noarch: erase skipped\n  Cleanup    : docker-engine-1.9.1-1.el7.centos.x86_64                      4/6 \n  Cleanup    : docker-engine-selinux-1.10.0-1.el7.centos.noarch             5/6 \n  Verifying  : docker-engine-1.10.1-1.el7.centos.x86_64                     1/6 \n  Verifying  : docker-engine-selinux-1.10.1-1.el7.centos.noarch             2/6 \n  Verifying  : python-requests-2.7.0-1.el7.noarch                           3/6 \n  Verifying  : python-requests-2.6.0-1.el7_1.noarch                         4/6 \n  Verifying  : docker-engine-selinux-1.10.0-1.el7.centos.noarch             5/6 \n  Verifying  : docker-engine-1.9.1-1.el7.centos.x86_64                      6/6 \n\nUpdated:\n  docker-engine.x86_64 0:1.10.1-1.el7.centos                                    \n  docker-engine-selinux.noarch 0:1.10.1-1.el7.centos                            \n\nFailed:\n  python-requests.noarch 0:2.6.0-1.el7_1  python-requests.noarch 0:2.7.0-1.el7 \n\nComplete!\n"]}
msg: Error unpacking rpm package python-requests-2.7.0-1.el7.noarch
python-requests-2.6.0-1.el7_1.noarch was supposed to be removed but is not!